src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Debug/DebugProfileEnumValuesGeneratorTests.cs

```diff
@@ -54,8 +59,13 @@ public class DebugProfileEnumValuesGenerator_Tests
 
             Mock<ILaunchSettingsProvider> moqProfileProvider = new Mock<ILaunchSettingsProvider>();
             moqProfileProvider.Setup(p => p.CurrentSnapshot).Returns(testProfiles.Object);
+            var moqThreadingService = new Mock<IProjectThreadingService>();
+            moqThreadingService
+                .Setup(p => p.JoinableTaskFactory)
+                .Returns(new Threading.JoinableTaskFactory(new Threading.JoinableTaskContext()));
```

Comment:
Hmmm... It seems to me you might want to have a shared utility method to create this mock so you're not doing it repeatedly.